manifests/s/SagerNet/sing-box/1.8.5/SagerNet.sing-box.installer.yaml

```
@@ -0,0 +1,29 @@
+# Created with komac v2.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: SagerNet.sing-box
+PackageVersion: 1.8.5
+InstallerType: zip
+NestedInstallerType: portable
+ReleaseDate: 2024-02-02
+Installers:
+- Architecture: x86
+  NestedInstallerFiles:
+  - RelativeFilePath: sing-box-1.8.4-windows-386\sing-box.exe
+    PortableCommandAlias: sing-box
+  InstallerUrl: https://github.com/SagerNet/sing-box/releases/download/v1.8.5/sing-box-1.8.5-windows-386.zip
+  InstallerSha256: 5595C2CE3E70E4024D5A29791F8769EB811D6EEF3C8BCB8266259C3E5B28D839
+- Architecture: x64
+  NestedInstallerFiles:
+  - RelativeFilePath: sing-box-1.8.4-windows-amd64\sing-box.exe
+    PortableCommandAlias: sing-box
+  InstallerUrl: https://github.com/SagerNet/sing-box/releases/download/v1.8.5/sing-box-1.8.5-windows-amd64.zip
+  InstallerSha256: 75E089E3742C153D72C214627EFD16AEF4CC493CB48BBD7996BB94EA6F2B05ED
+- Architecture: arm64
+  NestedInstallerFiles:
+  - RelativeFilePath: sing-box-1.8.4-windows-arm64\sing-box.exe

```

```suggestion
  - RelativeFilePath: sing-box-1.8.5-windows-386\sing-box.exe
    PortableCommandAlias: sing-box
  InstallerUrl: https://github.com/SagerNet/sing-box/releases/download/v1.8.5/sing-box-1.8.5-windows-386.zip
  InstallerSha256: 5595C2CE3E70E4024D5A29791F8769EB811D6EEF3C8BCB8266259C3E5B28D839
- Architecture: x64
  NestedInstallerFiles:
  - RelativeFilePath: sing-box-1.8.5-windows-amd64\sing-box.exe
    PortableCommandAlias: sing-box
  InstallerUrl: https://github.com/SagerNet/sing-box/releases/download/v1.8.5/sing-box-1.8.5-windows-amd64.zip
  InstallerSha256: 75E089E3742C153D72C214627EFD16AEF4CC493CB48BBD7996BB94EA6F2B05ED
- Architecture: arm64
  NestedInstallerFiles:
  - RelativeFilePath: sing-box-1.8.5-windows-arm64\sing-box.exe
```
Updating `RelativeFilePath` to match the updated `PackageVersion`.